packages/amazon/src/aws.validators.ts

```diff
@@ -1,18 +1,17 @@
 import { isEmpty } from 'lodash';
 
 export const iamRoleValidator = (value: string, label: string) => {
-  const isIAMRole = value.match(/^arn:aws:iam::\d{12}:role\/?\/[a-zA-Z_0-9+=,.@\-_/]+/);
+  const isIAMRole = value.match(/^arn:aws:iam::\d{12}:role\/?\/[a-zA-Z_0-9+=,.@\-/]+/);
```

Comment:
Doesn't [the plugin](https://github.com/spinnaker-plugins/aws-lambda-deployment-plugin-spinnaker/blob/ae9aca40cfce86d43b0d372939ae80e1a45a82ba/lambda-deployment-deck/src/utils/aws.validators.ts#L5) have the old value (with the underscore) here?

Reply:
It does, and I'm no Regex expert, but the underscore is already included after A-Z.

Reply:
Look-y there, so it is.